backend/app/services/splunk.py

```diff
@@ -110,6 +109,60 @@ async def _stream_results(self, oneshotsearch_results):
         for record in results.JSONResultsReader(oneshotsearch_results):
             yield record
 
+    async def filterPost(self, query, searchList=""):
+        """
+        Query data to construct filter from splunk server using splunk lib sdk
+
+        Args:
+            query (string): splunk query
+            OPTIONAL: searchList (string): additional query parameters for index
+        """
+
+        try:
+            # If additional search parameters are provided, include those in searchindex
+            searchindex = (
+                "search index={} {}".format(self.indice, searchList)
```

Comment:
This would look nicer as an f-string, `f"search index={self.indice} {searchList}"`

Reply:
Gotcha!